config.json

```diff
@@ -200,6 +200,26 @@
                 "MAX_LIMIT": 2,
                 "WINDOW_SIZE": 1440
             }
+        },
+        {
+            "ID": "2YBPw8vxrWQoVKLHooavpnCQej8yCpgK2EfqGQ8biELyfYfHfo",
+            "NAME": "Staava Subnet Fuji",
+            "TOKEN": "tSTAAV",
+            "RPC": "https://testnet-node.staava.network/",
+            "CHAINID": 603995,
+            "EXPLORER": "https://testnet.staava.network",
+            "IMAGE": "https://raw.githubusercontent.com/STAAVA/public/main/StaavaMainLogo.png",
+            "MAX_PRIORITY_FEE": "10000000000",
+            "MAX_FEE": "100000000000",
+            "DRIP_AMOUNT": 2,
+            "DECIMALS": 18,
+            "RECALIBRATE": 30,
+            "COUPON_REQUIRED": true,
```

Comment:
We currently do not support coupon or mainnet check service for Subnets. Would be great if you could remove these configs 👍